examples/misc/lib/effective_dart/usage_good.dart

```diff
@@ -396,6 +426,20 @@ class Point2 {
 }
 // #enddocregion semicolon-for-empty-body
 
+// #docregion no-new
+Widget build(BuildContext context) {
+  return new Row(
+    children: [
+      new RaisedButton(
+        onPressed: increment,
+        child: new Text('Increment'),
+      ),
+      new Text('Count: $counter'),
+    ],
+  );
+}
+// #enddocregion no-new
```

Comment:
Consider adding at least a FIXME note to actually remove the `new` keyword instances once it is supported (or place this example in the `..._bad.dart` file :)

Reply:
Done.